lib/ansispan.rb

```diff
@@ -1,4 +1,14 @@
 class Ansispan
+  @@escape_sequence = '\033'
+
+  def self.escape_sequence
+    @@escape_sequence
+  end
+
+  def self.escape_sequence=(custom_escape_sequence)
+    @@escape_sequence = Regexp.escape(custom_escape_sequence)
+  end
```

Comment:
I feel like the most common usage of this gem will just be using it for one conversion and not using it multiple times (unless maybe you're in a loop). With that in mind, do you think it makes more sense to add `escape_sequence` as an optional param to `Ansispan.convert`?